tests/Unit/SymfonyCache/TagSubscriberTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace FOS\HttpCache\Tests\Unit\SymfonyCache;
+
+use Symfony\Component\HttpFoundation\Request;
+use Prophecy\Argument;
+use FOS\HttpCache\SymfonyCache\TagSubscriber;
+use Symfony\Component\HttpFoundation\Response;
+
+class TagSubscriberTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var TagManagerInterface
+     */
+    private $tagManager;
+
+    /**
+     * @var CacheEvent
+     */
+    private $event;
+
+    public function setUp()
+    {
+        $this->tagManager = $this->prophesize('FOS\HttpCache\SymfonyCache\Tag\ManagerInterface');
```

Comment:
Would you be okay with changing this to Mockery? That’s what we now use in this library’s test suites. It may be a bit confusing to have different ways of mocking.